addons/halo/templates/configconstraint.yaml

```diff
@@ -1,19 +1,54 @@
+{{- $cc := .Files.Get "config/halo-config-effect-scope.yaml" | fromYaml }}
 apiVersion: apps.kubeblocks.io/v1alpha1
 kind: ConfigConstraint
 metadata:
   name: halo-cc
   labels:
     {{- include "halo.labels" . | nindent 4 }}
 spec:
+  reloadOptions:
```

Comment:
there is no `dynamicParameter` defined , there is no need to config `reloadOptions`, which is to trigger dynamic parameters wthout RESTART. Just remove it.

Reply:
got it